### PR TITLE
Some doctests for Sockets

### PR DESCRIPTION
**Pull request by kshyatt**
I wasn't sure about the local machine's IP address being the same for the buildbots or CI so that's why it's `julia-repl`.

### CODE REVIEW

File: stdlib/Sockets/src/addrinfo.jl [hunk 1]
```diff
@@ -46,7 +46,15 @@ end
...
 
 Gets all of the IP addresses of the `host`.
-Uses the operating system's underlying getaddrinfo implementation, which may do a DNS lookup.
+Uses the operating system's underlying `getaddrinfo` implementation, which may do a DNS lookup.
+
+# Example
+```jldoctest
+julia> getalladdrinfo("google.com")
+2-element Array{IPAddr,1}:
+ ip"172.217.6.174"
```
Comment (vtjnash) [inline stdlib/Sockets/src/addrinfo.jl]: google has a lot of IP addresses. `dig` / `host` / `nslookup` seem to bounce between at least 2 or 3 for me over a a short time window.  Perhaps a subdomain (like `ns1.google.com`) or a smaller site / one that we control might be a reliable enough choice?
```
julia> getalladdrinfo("julialang.org")
8-element Array{Sockets.IPAddr,1}:
 ip"151.101.194.49"    
 ip"151.101.130.49"    
 ip"151.101.66.49"     
 ip"151.101.2.49"      
 ip"2a04:4e42::561"    
 ip"2a04:4e42:400::561"
 ip"2a04:4e42:600::561"
 ip"2a04:4e42:200::561"
```

**Comment by fredrikekre** [comment]
Unfortunate that this makes the doctesting require internet access IMO (or do we have doctests that require that already? I don't think so). Perhaps better make them all `julia-repl`?

**Comment by kshyatt** [comment]
Yeah `julia-repl` might be a better choice...

**Comment by kshyatt** [comment]
Is this ok to go?

**Pull request merged**